controllers/hbase_controller.go

```diff
@@ -640,10 +640,25 @@ const (
 	HBaseControllerRevisionKey = "hbase-controller-revision"
 )
 
+var (
+	ignoreTemplateMetadataAnnotations = map[string]bool{
+		"kubectl.kubernetes.io/last-applied-configuration": true,
```

Comment:
nit: use `struct{}{}` when you want to represent a set in go. Bool still takes 1 byte of memory whereas `struct{}{}` doesn't take any space. In this case it obviously doesn't matter, but it's a good practice to have.

Reply:
Ah yeah... noticed the detail after - really done this time around